docs/src/main/asciidoc/native-and-ssl.adoc

```diff
@@ -157,14 +164,15 @@ Now, let's change the REST service URL to **not** use SSL in `src/main/resources
 
 [source,properties]
 ----
-quarkus.rest-client."org.acme.restclient.CountriesService".url=http://restcountries.eu/rest
+quarkus.rest-client."org.acme.rest.client.ExtensionsService".url=http://stage.code.quarkus.io/api
 ----
+and since http://stage.code.quarkus.io/api responds with 302 status code we need to also skip the tests with `-DskipTests`.
 
-And build again:
+Now we can build again:
 
 [source,bash]
 ----
-./mvnw clean install -Pnative
+./mvnw clean install -Pnative -DskipTests
```

Comment:
if wiremock is not removed, there's no need to skip tests

Reply:
So I have native SSL rework on my TODO list after this but didn't want to have unrelated changes in already bigger PR. The problem is that http://restcountries.eu/rest will respond with 302 Found which will fail the tests. Not sure if that should act as restcountries.eu where I suppose there was a redirect to https variant so the tests passed. A few possible solutions but I'm not sure which one would be the best. Should I adjust rest-client tests to expect either 200 and correct JSONs or 302 and no JSON?

Reply:
please ping me on Zulip when you get to implement that :)